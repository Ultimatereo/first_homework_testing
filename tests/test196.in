3
1e-07 5.2e-08 5e-07 3.5e-07
5.9e-08 4.11e-07 5e-09 2.79e-07
3.61e-07 3.03e-07 2.24e-07 2.47e-07
