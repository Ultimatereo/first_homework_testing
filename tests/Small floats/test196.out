-0.352494
0.720972
0.695518
